torch/nn/functional.py

```diff
@@ -3293,10 +3294,11 @@ def grid_sample(input, grid, mode='bilinear', padding_mode='zeros', align_corner
         grid (Tensor): flow-field of shape :math:`(N, H_\text{out}, W_\text{out}, 2)` (4-D case)
                        or :math:`(N, D_\text{out}, H_\text{out}, W_\text{out}, 3)` (5-D case)
         mode (str): interpolation mode to calculate output values
-            ``'bilinear'`` | ``'nearest'``. Default: ``'bilinear'``
+            ``'bilinear'`` | ``'nearest'`` | ``'bicubic'``. Default: ``'bilinear'``
             Note: When ``mode='bilinear'`` and the input is 5-D, the interpolation mode
             used internally will actually be trilinear. However, when the input is 4-D,
-            the interpolation mode will legitimately be bilinear.
+            the interpolation mode will legitimately be bilinear. 
```

Comment:
Would you make a slight change to this and the existing documentation for me? I've never liked it.

Instead of the current Note, can we replace it with:

Note: \`\`mode='bicubic'\`\` supports only 4-D input. \`\`mode='bilinear'\`\` performs bilinear interpolation on 4-D inputs, but trilinear interpolation on 5-D inputs.

Reply:
In the begining, I tried to append the 4-d input warning and more information to the existed documentation, but found it made the whole document too log. Hence, I decide to only append the warning message(most important part) and put the remaining to a new "Note" block. Sorry that makes the modification seems careless. Your suggestion is more consice.